EnglishFilter/sections/antiadblock.txt

```diff
@@ -1698,6 +1698,8 @@ pirate.ws#%#window.spoof_weer2edasfgeefzc = true;
 ! https://github.com/AdguardTeam/AdguardFilters/issues/27518
 !+ PLATFORM(ext_ff, ext_opera, ios, ext_android_cb, ext_ublock)
 @@||uploadproper.com^$generichide
+! https://github.com/AdguardTeam/AdguardFilters/issues/36048
+@@||textfree.us^$generichide
```

Comment:
What is the rule fixing? Issue itself is not giving any useful information.

Reply:
Anti-adblock after completed registration. No ads here, just anti-adblock.

Reply:
OK, approved. Sadly I could not register on the site.